docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
```

Comment:
Well, to me it's a functional testing topic, cause: Unit testing has nothing to do with URLs at all. And in Acceptance testing, there's no need for Guzzle, since you have a browser anyway. Right? => So Functional is the only one which is left ;-)

Didn't know that a "Cookbook" exists ;-) But I must say: One of the main problems of the docs is that they are scattered around 200 places - each with different layout, different wording, different commit policy. So if you have a question, you cannot go to one place and look for an answer, but you have to figure out first which place is the right to start looking.  
So I'd suggest to integrate those 7 cookbook articles into the main docs: The 2 WebDriver articles to the WebDriver Module, the 1 Symfony article to the Symfony Module, and so on...